content/app/github/wrappers/line-element-wrapper.js

```diff
@@ -11,9 +11,46 @@ define("content/app/github/wrappers/line-element-wrapper", function(require, exp
 		$statics : {
 
 			TAG_NAME : 'tr',
+			TAG_LINE_CLASS : '',
+			TAG_EXPANDABLE_BUTTON_CLASS : 'js-expandable-line',
 			NUMBER_ATTR_NAME : 'data-line-number',
 
 			/**
+			 * Populate missing line ranges.
+			 * This should set ranges for expand button lines.
+			 * @param  {LineElementWrapper[]} lines
+			 */
+			_populateMissingRanges : function (lines) {
+
+				for (var i = 0; i < lines.length; i++) {
+					var line = lines[i];
+
+					if (line.isExpandButton) {
+						var prevOldLineNumber = 0;
+						var nextOldLineNumber = Number.MAX_VALUE;
+						var prevNewLineNumber = 0;
+						var nextNewLineNumber = Number.MAX_VALUE;
+
+						var prevLine = lines[i-1];
+						if (prevLine) {
+							prevOldLineNumber = prevLine.oldNumberRange[1];
+							prevNewLineNumber = prevLine.newNumberRange[1];
+						}
+
+						var nextLine = lines[i+1];
+						if (nextLine) {
+							nextOldLineNumber = nextLine.oldNumberRange[0];
+							nextNewLineNumber = nextLine.newNumberRange[0];
+						}
+
+						line.oldNumberRange = [prevOldLineNumber, nextOldLineNumber];
```

Comment:
Should not those values be increased / decreased by 1? Since oldNumber range starts from prevOldLineNumber + 1?